packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -124,4 +127,51 @@ export class PipelineService {
       buttons: [Dialog.okButton()]
     });
   }
+
+  static convertPipeline(pipelineDefinition: any): any {
```

Comment:
Per previous comment. 
```suggestion
  static convertPipeline(pipelineDefinition: any, currentVersion: number): any {
```